api/controllers.js

```diff
@@ -49,7 +49,21 @@ module.exports = {
       res.status(400).send('MISSING INPUT - senderId, userId2, and (body or photo) are required');
       return;
     }
-    db.createNewConversation(req.body)
+    console.log(req.body);
+    return helpers.storePhoto(req)
+      .then((photoData) => {
+        let conversation = {};
+        if (photoData != null) {
+          conversation.senderId = req.body.senderId;
+          conversation.userId2 = req.body.userId2;
+          conversation.photo = photoData.Location;
```

Comment:
is there a reason why Location is capitalized?

Reply:
That is how s3 sends the data back. All of the keys are capitalized.